components/brave_news/common/brave_news.mojom

```diff
@@ -140,6 +141,37 @@ struct Publisher {
   UserEnabled user_enabled_status = UserEnabled.NOT_MODIFIED;
 };
 
+struct HeroArticle {
```

Comment:
Q. I'm curious what HeroArticle is. Is it something like featured or highlighted?

Reply:
Yeah, basically an article but with a bigger image. I think something like the big article here (the smaller ones being `Article`):
![image](https://github.com/brave/brave-core/assets/7678024/e0032e0c-924b-425e-8680-8209b1ddf621)

My current component on the frontend looks pretty much identical though :laughing: